src/modules/http.js

```diff
@@ -0,0 +1,78 @@
+const axios = require('axios')
+const { get } = require('lodash')
+
+class APIClient {
+
+    constructor(queryApiUrl, publishApiUrl) {
+        const subpath = '/api/v1'
+        this.queryApiUrl = queryApiUrl + subpath
+        this.publishApiUrl = publishApiUrl + subpath
+        this.axios = axios.create()
+    }
+
+    async getPublisherDID() {
+        const response = await this.axios.get(`${this.publishApiUrl}/id`)
+        const did = get(response, 'data.did', null)
+        return did
+    }
+
+    async readReviewRecord(multihash, options = {}) {
+        const response = await this.axios.get(`${this.queryApiUrl}/reviews/${multihash}`, {
+            params: {
+                getLatestVersion: get(options, 'getLatestVersion', true)
+            }
+        })
+        return response.data
+    }
+
+    async storeReviewRecord(reviewRecord, opt = {}) {
+        const options = Object.assign({}, {
+            publish: true
+        }, opt)
+        const response = await this.axios.post(`${this.publishApiUrl}/reviews`, reviewRecord, {
+            params: {
+                publish: get(options, 'publish', true),
+                bitcoinTransactionHash: get(options, 'bitcoinTransactionHash', ''),
+                expectedMultihash: get(options, 'expectedMultihash', '')
+            }
+        })
+        return response.data
+
+    }
+
+    async getReviewsWrittenByDID(didId, offset = 0, limit = 0) {
+        const response = await this.axios.get(`${this.queryApiUrl}/dids/${didId}/reviews/writtenby`, {
+            params: { limit, offset }
+        })
+        return response.data
+    }
+
+    async getReviewsAboutDID(didId, offset = 0, limit = 0) {
+        const response = await this.axios.get(`${this.queryApiUrl}/dids/${didId}/reviews/about`, {
+            params: { limit, offset }
+        })
+        return response.data
+    }
+
+    async getDID(didId, waitUntilPresent = false) {
+        const response = await this.axios.get(`${this.queryApiUrl}/dids/${didId}`, {
+            params: {
+                waitUntilPresent: Boolean(waitUntilPresent)
+            }
+        })
+        return response.data
+    }
+
+    async getPoPR(multihash) {
```

Comment:
Remove?

Reply:
We need this in the wallet. Or we can have the wallet get it from IPFS, what do you think is best?